app/models/piece.rb

```diff
@@ -18,6 +18,27 @@ def move!(x,y)
     end
   end
 
+  #Inital Capture Logic:
+  #Need game model to test id of game
+
+  #Set initial captured status to false
+  def default_captured_status
+    self.captured = false if self.captured = nil
+  end
+
+  #Find the target piece in the database base on the x,y coordinates
+  def capturable?(x,y)
+    Piece.where(:x_position => x, :y_position => y, :captured => false).where.not(:color => color).present?
```

Comment:
`:x_position => x` has updated rails syntax logic and can be more simply expressed as `x_position: x`.